client/components/happychat/style.scss

```diff
@@ -136,6 +145,83 @@
 
 
 /**
+ * Minimize animation
+ */
+
+.layout .happychat .happychat__container.is-open.minimizing {
+	box-shadow: none;
+	border-top: 1px solid lighten( $gray, 20% );
+	border-left: 1px solid lighten( $gray, 20% );
+	border-right: 1px solid lighten( $gray, 20% );
+	background: lighten( $gray, 30% );
+	animation: minimize .5s 1 forwards;
+
+	> .happychat__loading,
+	> .happychat__composer,
+	> .happychat__welcome,
+	> .happychat__conversation,
+	> .happychat__title .happychat__active-toolbar * {
+		visibility: hidden;
+	}
+
+	> .happychat__title {
+		transform: translateZ(0);
+		position: relative;
+		animation: disappear .1s 1 forwards;
+	}
+
+}
+
+@keyframes disappear {
```

Comment:
Hmm, that's a good question. I don't know if it makes sense — the animation and `@keyframes` used by the animation are highly specific and not very conducive to reuse at all: https://github.com/Automattic/wp-calypso/pull/8706/commits/cd73d2cc3f9bab3579f677c3686845c3fd90708c#diff-7bac90f26854177f495f0c4de694de08R187

They contain specific coordinates from where the animation starts and where it ends. Or perhaps my understanding that the component is only for reusable animations is wrong?